tests/console/yast2_dns_server_service_abort_keep.pm

```diff
@@ -0,0 +1,36 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Ensure the service keep inactive & disabled after set on tree-based
+# interface and abort to accept the setting for yast2 dns-server
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+use base "y2_module_consoletest";
+use strict;
+use warnings;
+use testapi qw(save_screenshot select_console);
+use utils;
+
+sub run {
+    my $dnsserver = $testapi::distri->get_dns_server();
+
+    select_console 'root-console';
+    YaST::Module::open(module => 'dns-server', ui => 'ncurses');
+
+    $dnsserver->process_reading_configuration();
```

Comment:
let's `save_screenshot` here as well (if we catch the screen on time, of course, otherwise doesn't make sense...). Apply in the other test cases if works, please.

Reply:
could you apply in the rest of the places?